example/sonik/app/routes/_layout.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, StyleRegistry, createStyleRegistry } from "@kuma-ui/core";
+import type { LayoutHandler } from "@sonikjs/react";
+import { renderToString } from "react-dom/server";
+
+declare const __KUMA_CSS__: string; // inserted by vite plugin
+
+const handler: LayoutHandler = ({ children, head }) => {
+  const body = (
+    <body>
+      <div className="wrapper">
+        <header>
+          <h1>
+            <a href="/">Top</a>
+          </h1>
+        </header>
+        <Box color="green" fontWeight="bold" fontSize="24px" py="16px">
+          This is colored by Kuma UI
+        </Box>
+        {children}
+        <footer style={{ marginTop: "2rem" }}>
+          <small>© 2023 your name</small>
+        </footer>
+      </div>
+    </body>
+  );
+
+  const registry = createStyleRegistry();
+  renderToString(<StyleRegistry registry={registry}>{body}</StyleRegistry>); // Render here to determine the initial CSS.
+  const runtimeCss = registry.styles();
+  registry.flush();
```

Comment:
:memo: Sonik doesn't have a feature that equivalent to `useServerInsertedHTML` so render twice like in next.js Pages Router